### DISCUSSION TITLE
Parquet: support Parquet FIXED_LENGTH_BYTE_ARRAY mapping to FixedString

### DISCUSSION THREAD

**Discussion by vbmithr** [ClickHouse/ClickHouse#45324] [Ideas]
I see that it is not supported. Any idea why? It seems that there is a perfect fit between the Parquet and Clickhouse type.

**Comment by den-crane** [comment]
I guess it's accidentally.

@Avogar

**Comment by Avogar** [comment]
Yes, will be easy to add. I'll do it: https://github.com/ClickHouse/ClickHouse/issues/45326

**Reply by vbmithr** [reply]
Thanks guys.
